Add Staking template docs (#530)
aptos-labs/developer-docs: apps/nextra/pages/en/build/create-aptos-dapp/templates/token-minting-dapp.mdx

3. My Fungible Assets page - A page to view all the assets created under the current Move module (smart contract). This page is not accessible on production.

## Quick Start
- [Create a token on chain](#publish-the-move-contract)
- [Customizing the frontend](#customizing-the-front-end)
- [Deploy to a live server](#deploy-to-a-live-server)

## Setup

You can start the app by running:

```bash 
npm run dev
```

However, for the app to work as intended, there are several setup steps you will have to do to:

1. Publish the contract and create the token on-chain.
2. Config your dapp to use the token

## Publish the Move Contract

<Steps>

### On the project root folder, use the below command to initiliaze an account to publish the contract with.

> ```bash 

Reviewer: ```suggestion
```bash filename="Terminal"
```

npm run move:init
```

The CLI will prompt you to provide the private key for your wallet in order for the tool to publish the contract on your behalf. If you provide no input, it will generate one for you.

> This will create an account you will later publish the contract with, as well as setting the `VITE_MODULE_ADDRESS` in the `.env` file to set the contract object address.

Reviewer: ```suggestion
This will:
1. Create an account you will later publish the contract with
2. Set the `VITE_MODULE_ADDRESS` in the `.env` file to the contract object address.
```

### In the `.env` file, set `VITE_FA_CREATOR_ADDRESS` to be the same as your account address.
This determines which account is allowed to create assets using the dapp.

If you haven't set up an account, you can create one with an Aptos wallet like [Petra](https://petra.app/docs/use#create-a-new-account). Make sure to set up the account on the same network as your template project.

For example:
```bash filename=".env"
VITE_APP_NETWORK=testnet
VITE_FA_CREATOR_ADDRESS=0x71e3cdf1ef86d575ae38c7c2d19f22b8c88eb2b15082691782e5b8c8da401568

> ```

Reviewer: nice 👍

### On the project root folder, run the below command to publish the contract.

> ```bash

Reviewer: ```suggestion
```bash filename="Terminal"
```

npm run move:publish
```
<Callout type="info">
You will have to confirm the gas fee in the console.
</Callout>

This command will publish the contract to chain.

</Steps>

## Publishing the asset on-chain
<Steps>

### Run the app and open the preview.

> ```bash

Reviewer: ```suggestion
```bash filename="Terminal"
```

npm run dev
```

### In the top-right corner, click "Connect a Wallet". 
Use the same account that you set in `.env` for the `VITE_FA_CREATOR_ADDRESS`. 
Or, if you use a different account, set the `VITE_FA_CREATOR_ADDRESS` to the account you sign in with and republish the contract.

<Callout type="warning">

> Do not use the "Google Account" option if it is available. The account generated from that will not work as the `VITE_COLLECTION_CREATOR_ADDRESS`. Use a wallet created with an Aptos wallet provider such as [Petra](https://petra.app/docs/use#create-a-new-account).

Reviewer: I'm a little confused with what this means

Author: simply says you can't create a collection with a Aptos Connect account
